### PR TITLE
Enable appfilter and landing nav on all environments

### PR DESCRIPTION
**Pull request by karelhala**

### CODE REVIEW

File: src/js/App/Header/Header.js [hunk 1]
```diff
@@ -20,7 +18,7 @@ export const Header = ({ logoClassName }) => {
       <a href="./" className={`ins-c-header-link${logoClassName ? ` ${logoClassName}` : ''}`}>
         <Logo />
       </a>
-      {user && isFilterEnabled && <AppFilter />}
+      <AppFilter />
```
Comment (PanSpagetka) [inline src/js/App/Header/Header.js:21]: Do we want to show it without `user`?

**Review by PanSpagetka — APPROVED** [review]

File: src/js/App/Header/Header.js (hunk 1 above)
Comment (Hyperkid123) [reply src/js/App/Header/Header.js:21]: @karelhala I don't think so. Maybe its for some special openshift use case?

**Comment by karelhala** [comment]
Restarting travis

File: src/js/App/Header/Header.js (hunk 1 above)
Comment (karelhala) [reply src/js/App/Header/Header.js]: It was a mistake, the filter should be visible only for logged in users.

**Pull request merged**